appengine/flexible/memcache/app.yaml

```diff
@@ -1,6 +1,9 @@
 runtime: python
-vm: true
+env: flex
 entrypoint: gunicorn -b :$PORT main:app
 
 runtime_config:
   python_version: 3
+
+beta_settings:
```

Comment:
Can you add a comment here describing why this is necessary? Maybe a link to the flex memcache docs page that will describe the process for signing up for whitelisting?